### DISCUSSION TITLE
Jira에서 여러 개의 Sprint를 동시에 진행할 수 있게 되었네요?

### DISCUSSION THREAD

**Discussion by Woon-2** [MyeongWoonJang/cpp-study#24] [Q&A] [answered]
그 전에는 백로그에서 스프린트를 만들어도 스프린트 시작 버튼이 활성화되지 않았던 것 같은데,   
어떻게 된 일인가요??

**Comment by Lcat12345** [comment, marked as answer]
스프린트 만들기
---
<div align="center"><image width="600" height="400" src="https://user-images.githubusercontent.com/62102830/193520064-f569e045-1e47-4708-b571-ea804db667fb.png"></div>  

이슈가 없을 땐 스프린트를 시작할 수 없습니다.
![image](https://user-images.githubusercontent.com/62102830/193520921-d4693e81-fca8-45ce-b13d-024b51c49cfe.png)

---
백로그에서 스프린트로 이슈 추가
---
![image](https://user-images.githubusercontent.com/62102830/193521222-9136a325-1a89-44bc-b4b9-4f56b6f15253.png)

![image](https://user-images.githubusercontent.com/62102830/193521385-acfb08c6-0928-481e-98e2-b23ac3a32022.png)

![image](https://user-images.githubusercontent.com/62102830/193521459-dcbd4556-87d2-4d6f-91ec-e0866bfd4cd0.png)

아마 다른 사람이 만든 스프린트라 시작이 활성이 안됫나 추측해봅니다.

**Reply by Woon-2** [reply]
오호 그렇군요. 참고해야겠습니다.
